server/services/licenceService.js

```diff
@@ -106,6 +107,18 @@ module.exports = function createLicenceService(licenceClient, establishmentsClie
         }
     }
 
+    async function updateBassReferral(data = {}) {
+        try {
+            const nomisId = data.nomisId;
+            const bassReferralData = createBassReferralObject(data);
+
+            return await licenceClient.updateSection('bassReferral', nomisId, bassReferralData);
+        } catch (error) {
+            console.error('Error during setBassReferral', error.stack);
+            throw error;
+        }
+    }
+
     async function sendToOmu(nomisId) {
         try {
             return await licenceClient.updateStatus(nomisId, 'SENT');
```

Comment:
while your here since you can go ahead and remove async from this function since it's not needed

Reply:
I deleted all this stuff, not sure why it's back in